### ISSUE TITLE
Merging Companies

### ISSUE THREAD

**Issue by adsherpa** [agilecrm/rest-api#66]
Is there a way to merge two or more companies that exist as duplicates?
This should put all contacts to one company

**Comment by krishna-prasadh** [comment]
Yes, you can merge duplicate companies. Open the company you want to merge. You can find menu under the company name and select Merge Duplicates option from menu. This will show all the duplicate records you can merge them by mapping the fields. This will also merge the contacts of the companies. So that you can see all the contacts under one company. Once you merge duplicates you can't revert them back.
